modules/virt-cluster-resource-requirements.adoc

```diff
@@ -49,16 +53,21 @@ Calculate the cluster processor overhead requirements for {VirtProductName} by u
 .Cluster CPU overhead
 
 ----
-CPU overhead for infrastructure nodes ≈ 4
+CPU overhead for infrastructure nodes ≈ 4 cores
 ----
 
+{VirtProductName} increases the overall utilization of cluster level services such as logging, routing, and monitoring. To account for this workload, ensure that nodes hosting infrastructure components have capacity allocated for 4 additional cores (4000 millicores) across all nodes.
```

Comment:
s/nodes hosting/nodes that host

This might just be me, but I'm tripped up by "across all nodes." Can you clarify what this means?

Reply:
From what I understand, it means that the overhead is 4 additional cores that are distributed across all nodes that host infrastructure components and not 4 cores for each node.